cypress/support/commands.js

```diff
@@ -0,0 +1,32 @@
+import waitForDom from "./waitForDom"
+
+Cypress.Commands.overwrite("visit", (originalFn, url, options) => {
+  // originalFn is the existing `visit` command that you need to call
+  // and it will receive whatever you pass in here.
+  //
+  // make sure to add a return here!
+  originalFn(url, options)
+  cy.wait(2 * 1000)
+  return cy.get(`[data-testid="verify-user-modal"]`).should("not.exist")
```

Comment:
could i double-check on why we are over-writing the `visit` command for cypress? it doesn't seem to be required and it seems to impose an additional 2s lag on every `visit`

Reply:
This is mostly for the assertion that the user modal doesn't exist before proceeding! I included the wait in case the modal took some time to appear, but can remove that if necessary

Reply:
hmm i think this might be worth reconsidering as we currently have ~50 invocations of `visit` in our test suite and the modal doesn't seem to be interfering w/ our e2e anyway

Reply:
As discussed offline, we're going to remove this override and just monitor for now - we can come back to this in future if the modal not disappearing after a visit is a consistent problem